File: sab.py
require"pwm"
import requests
a = PWM.new(16)
import time
sp=0
a.frequency(5)
while True:
    dt=requests.get(random_int)#シガワが送ってきた値
    while sp < dt:
        sp=sp+1
        a.duty(sp)
        time.sleep(0.5)
    while sp > dt:
        sp=sp-1
        a.duty(sp)
        time.sleep(0.5)
    while sp == dt:
        a.duty(sp)

File: sab.rb
require"pwm"
a = PWM.new(16)
#超高速
a.frequency(5)
a.duty(50)
#高速
a.frequency(5)
a.duty(25)
#普通
a.frequency(5)
a.duty(10)
#遅い
a.frequency(5)
a.duty(6)